demo/config.yaml

```diff
@@ -20,3 +20,11 @@ lcr-modules:
         inputs:
             sample_bam: "data/{sample_id}.bam"
             sample_bai: "data/{sample_id}.bam.bai"
+
+    strelka:
+        inputs:
+            sample_bam: "data/{sample_id}.bam"
+            sample_bai: "data/{sample_id}.bam.bai"
+            candidate_small_indels_output: "results/manta-2.1/99-outputs/vcf/{seq_type}--{genome_build}/candidateSmallIndels/{tumour_id}--{normal_id}--{pair_status}.candidateSmallIndels.vcf"
+            candidate_small_indels_vcf: "results/manta-2.1/02-manta/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/results/variants/candidateSmallIndels.vcf.gz"
+            candidate_small_indels_tbi: "results/manta-2.1/02-manta/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/results/variants/candidateSmallIndels.vcf.gz.tbi"
```

Comment:
I thought you said that you can't directly point to the VCF files produced by Manta, no?

Reply:
I can, I just can't use it as an input because it would trigger a `MissingInputException`